report/finished/chapters/modeling.tex

```diff
@@ -1,6 +1,206 @@
+    \label{eqn:DH-transformation}
+\end{equation}
+that consists of the four basic transformations
+\begin{equation}
+    Rot_z(\theta_i) = 
+    \begin{bmatrix}
+        cos(\theta_i) & -sin(\theta_i) & 0 & 0 \\
+        sin(\theta_i) & cos(\theta_i) & 0 & 0 \\
+        0 & 0 & 1 & 0 \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}
+\end{equation}
+\begin{equation}
+    Trans_z(d_i) = 
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & 1 & 0 & 0 \\
+        0 & 0 & 1 & d_i \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}
+\end{equation}
+\begin{equation}
+    Trans_x(\alpha_i) = 
+    \begin{bmatrix}
+        1 & 0 & 0 & \alpha_i \\
+        0 & 1 & 0 & 0 \\
+        0 & 0 & 1 & 0 \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}
+\end{equation}
+\begin{equation}
+    Rot_z(\theta_i) = 
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & cos(\alpha_i) & -sin(\alpha_i) & 0 \\
+        0 & sin(\alpha_i) & cos(\alpha_i) & 0 \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}.
+\end{equation}
+
+
+Where the parameters \(\theta_i\), \(a_i\), \(d_i\) and \(\alpha_i\), known as DH-parameters, characterize each joint. To allow the transformation for each joint to be represented by only four parameters, compared to the six parameters that is required in general, there are some restrictions on how the coordinate frames can be chosen. To comply with the DH-convention, the following must be satisfied.
+\begin{itemize}
+    \item The axis \(x_{1}\) is perpendicular to the axis \(z_0\)
+    \item The axis \(x_{1}\) intersects the axis \(z_0\)
+\end{itemize}
+where it is assumed that two frames are given, frame 0 and frame 1, and the transformation from equation \ref{eqn:DH-transformation} transforms a coordinate from frame 1 into a coordinate in frame 0.\cite{spong}
+
+
+
+\subsection*{Kinematic Chain} %A1*A2*...
+A robotic manipulator can be described by a set of joints with links between them where a homogeneous transformation matrix \(A_i\) that describes the transformation with respect to the previous joint exists for each joint. This means that a transformation that describes the position and orientation of joint j with respect to a joint i to a can be found by a transformation matrix \cite{spong}
+\begin{equation}
+    \begin{cases}
+        T_j^i = A_{i+1}A_{i+2}...A_{j-1}A_{j}, \text{  if \(i < j\) } \\
+        T_j^i = I, \text{  if \(i = j\) } \\
+        T_j^i = (T_i^j)^{-1}, \text{  if \(i > j\) }
+    \end{cases}
+    \label{eqn:Kinematic_chain_spong}
+\end{equation}
+
+
+
+\subsection*{Forward Kinematics}
+Forward kinematics is the problem of finding the position and orientation of the end effector.
+
+Since a manipulator can be seen as a kinematic chain the problem of finding the position and orientation of the end effector can be solved by finding the transformation matrices in equation \ref{eqn:Kinematic_chain_spong} needed to solve for the transformation matrix for the final joint.
```

Comment:
The part after the \ref{} is not needed. Either remove it or rework the sentence

Reply:
yes, I will remove it.